lib/cocoapods-search/command/search.rb

```diff
@@ -73,8 +73,9 @@ def web_search
       end
 
       def local_search
-        query_regex = @query.join(' ').strip
-        query_regex = Regexp.escape(query_regex) unless @use_regex
+        query_regex = @query.reduce('') { |result, q|
```

Comment:
Here too you could use an `Array.new`, and `result << (@use_regex ? q : Regexp.escape(q))`, then `join` before stripping.